test/app/background/api/cosmeticFilterAPITest.ts

```diff
@@ -0,0 +1,263 @@
+
+import 'mocha'
+import * as sinon from 'sinon'
+import * as assert from 'assert'
+
+import * as cosmeticFilterAPI from '../../../../app/background/api/cosmeticFilterAPI'
+
+describe('cosmeticFilterTestSuite', () => {
+  describe('addSiteCosmeticFilter', function () {
+    const url = 'https://www.brave.com'
+    const filter = '#cssFilter'
+
+    let sampleValue: any
+    console.log(sampleValue)
```

Comment:
same as other console.log comments pls remove

Reply:
If you remove this, you get the following error:

```
> brave-extension@1.0.0 test-unit /Users/brianlee/Documents/brave-extension
> cross-env NODE_ENV=test TS_NODE_TYPE_CHECK=true mocha -r ignore-styles ./test/setupApp.ts test/app/**/**/**/*.{ts,tsx}


/Users/brianlee/Documents/brave-extension/node_modules/ts-node/src/index.ts:253
    return new TSError(diagnosticText, diagnosticCodes)
           ^
TSError: ⨯ Unable to compile TypeScript:
test/app/background/api/cosmeticFilterAPITest.ts(13,9): error TS6133: 'sampleValue' is declared but its value is never read.

    at createTSError (/Users/brianlee/Documents/brave-extension/node_modules/ts-node/src/index.ts:253:12)
    at getOutput (/Users/brianlee/Documents/brave-extension/node_modules/ts-node/src/index.ts:363:40)
    at Object.compile (/Users/brianlee/Documents/brave-extension/node_modules/ts-node/src/index.ts:551:11)
    at Module.m._compile (/Users/brianlee/Documents/brave-extension/node_modules/ts-node/src/index.ts:435:43)
    at Module._extensions..js (internal/modules/cjs/loader.js:713:10)
    at Object.require.extensions.(anonymous function) [as .ts] (/Users/brianlee/Documents/brave-extension/node_modules/ts-node/src/index.ts:438:12)
    at Module.load (internal/modules/cjs/loader.js:612:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)
    at Function.Module._load (internal/modules/cjs/loader.js:543:3)
    at Module.require (internal/modules/cjs/loader.js:650:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at /Users/brianlee/Documents/brave-extension/node_modules/mocha/lib/mocha.js:250:27
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (/Users/brianlee/Documents/brave-extension/node_modules/mocha/lib/mocha.js:247:14)
    at Mocha.run (/Users/brianlee/Documents/brave-extension/node_modules/mocha/lib/mocha.js:576:10)
    at Object.<anonymous> (/Users/brianlee/Documents/brave-extension/node_modules/mocha/bin/_mocha:637:18)
    at Module._compile (internal/modules/cjs/loader.js:702:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)
    at Module.load (internal/modules/cjs/loader.js:612:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)
    at Function.Module._load (internal/modules/cjs/loader.js:543:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:744:10)
    at startup (internal/bootstrap/node.js:240:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:564:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! brave-extension@1.0.0 test-unit: `cross-env NODE_ENV=test TS_NODE_TYPE_CHECK=true mocha -r ignore-styles ./test/setupApp.ts test/app/**/**/**/*.{ts,tsx}`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the brave-extension@1.0.0 test-unit script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

This one might have to stay - let me know if you have any suggestions on how to get this working because it has be called somewhere, the possibility that it is called is not enough to satisfy TypeScript.